I have contributed to the following publications:
* Gerber Z, Fisun M, Aschard H, **Djebali S**, 2024, [*PaintorPipe: a pipeline for genetic variant fine-mapping using functional annotations. Bioinformatics Advances*](https://academic.oup.com/bioinformaticsadvances/article/4/1/vbad188/7486556), 4(1), p.vbad188
* Kurylo C, Guyomar C, Foissac S, **Djebali S**, 2023, [*TAGADA: a scalable pipeline to improve genome annotations with RNA-seq data*](https://academic.oup.com/nargab/article/5/4/lqad089/7318076), NAR Genomics and Bioinformatics, 5(4), p.lqad089
* Eché C, Iampietro C, Birbes C, Dréau A, Kuchly C, Di Franco A, Klopp C, Faraut T, **Djebali S**, Castinel A and Zytnicki M, 2023, [*A Bos taurus sequencing methods benchmark for assembly, haplotyping, and variant calling*](https://www.nature.com/articles/s41597-023-02249-1), Scientific Data, 10(1), p.369
* Hoellinger H, Mestre C, Aschard H, Le Goff W, Foissac S, Faraut T, **Djebali S**, 2023, [*Enhancer/gene relationships: need for more reliable genome-wide reference sets*](https://www.frontiersin.org/journals/bioinformatics/articles/10.3389/fbinf.2023.1092853/full), Frontiers in Bioinformatics, 24(3):1092853
* Jehl F, Degalez F, Bernard M, Lecerf F, Lagoutte L, Désert C, Coulée M, Bouchez O, Leroux S, Abasht B, Tixier-Boichard M, Bed’hom B, Burlot T, Gourichon D, Bardou P, Acloque H, Foissac S, **Djebali S**, Giuffra E, Zerjal T, Pitel F, Klopp C, Lagarrigue S, 2021, [*RNA-Seq Data for Reliable SNP Detection and Genotype Calling: Interest for Coding Variant Characterization and Cis-Regulation Analysis by Allele-Specific*](https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2021.655707/full), Frontiers in Genetics, 12:655-707
* Robic A, Faraut T, Feve K, **Djebali S**, Prunier A, Larzul C, Liaubet L, 2021, [*Correlation networks provide new insights into the architecture of testicular steroid pathways in pigs*](https://www.mdpi.com/2073-4425/12/4/551), Genes, 12(4):551
* Moore JE, ... **Djebali S** ... , Weng Z, 2020, [*Expanded encyclopaedias of DNA elements in the human and mouse genomes*](https://www.nature.com/articles/s41586-020-2493-4), Nature, 583(7818):699-710
* Jehl F, Muret K, Bernard M, Boutin M, Lagoutte L, Désert C, Dehais P, Esquerré D, Acloque H, Giuffra E, **Djebali S**, Foissac S, Derrien T, Pitel F, Zerjal T, Klopp C, Lagarrigue S, 2020, [*An integrative atlas of chicken long non-coding genes and their annotations across 25 tissues*](https://www.nature.com/articles/s41598-020-77586-x), Scientific Reports, 20(1):1-17
* Breschi A, Muñoz-Aguirre M, Wucher V, Davis CA, Garrido-Martín D, **Djebali S**, Gillis J, Pervouchine DD, Vlasova A, Dobin A, Zaleski C, Drenkow J, Danyko C, Scavelli A, Reverter F, Snyder MP, Gingeras TR and Guigó R, 2020, [*A limited set of transcriptional programs define major cell types*](https://genome.cshlp.org/content/30/7/1047.short), Genome Research, 30(7):1047-1059
* Snyder MP, ... **Djebali S** ... , Myers RM, 2020, [*Perspectives on ENCODE*](https://www.nature.com/articles/s41586-020-2449-8), Nature, 583(7818):693-698
* Foissac* S, **Djebali S***, Munyard K, Vialaneix N, Rau A, Muret K, Esquerre D, Zytnicki M, Derrien T, Bardou P, Blanc F, Cabau C, Crisci E, Dhorne-Pollet S, Drouet F, Faraut T, Gonzales I, Goubil A, Lacroix-Lamande S, Laurent F, Marthey S, Marti-Marimon M, Momal-Leisenring R, Mompart F, Quéré P, Robelin D, San Cristobal M, Tosser-Klopp G, Vincent-Naulleau S, Fabre S, Pinard-Van der Laan M-H, Klopp C, Tixier-Boichard M, Acloque H, Lagarrigue S, Giuffra E, 2019, [*Multi-species annotation of transcriptome and chromatin structure in domesticated animals*](https://link.springer.com/article/10.1186/s12915-019-0726-5), BMC Biology, 17:1-25
* Giuffra E, Tuggle CK, Faang Consortium including **Djebali S**, 2019, [*Functional Annotation of Animal Genomes (FAANG): current achievements and roadmap*](https://www.annualreviews.org/content/journals/10.1146/annurev-animal-020518-114913), Annual review of animal biosciences,15(7):65-88
* Robic A, Faraut T, **Djebali S**, Weikard R, Feve K, Maman S, Kuehn C, 2019, [*Analysis of pig transcriptomes suggests a global regulation mechanism enabling temporary bursts of circular RNAs*](https://www.tandfonline.com/doi/full/10.1080/15476286.2019.1621621), RNA biology, 16(9):1190-1204
* Muret K, Klopp C, Wucher V, Esquerré D, Legeai F, Lecerf F, Désert C, Boutin M, Jehl F, Acloque H, Giuffra E, **Djebali S**, Foissac S, Derrien T, Lagarrigue S, 2017, [*Long noncoding RNA repertoire in chicken liver and adipose tissue*](https://link.springer.com/article/10.1186/s12711-016-0275-0), Genetics Selection Evolution, 49(1):6 
* Hon CC, Ramilowski JA, Harshbarger J, Bertin N, Rackham O, Gough J, Denisenko E, Schmeier S, Poulsen TM, Severin J, Lizio M, Kawaji H, Kasukawa T, Itoh M, Burroughs AM, Noma S, **Djebali S**, Alam T, Medvedeva YA, Testa AC, Lipovich L, Yip CW, Abugessaisa I, Mendez M, Hasegawa A, Tang D, Lassmann T, Heutink P, Babina M, Wells CA, Kojima S, Nakamura Y, Suzuki H, Daub C, de Hoon MJ, Arner E, Hayashizaki Y, Carninci P, Forrest AR, 2017, [*An atlas of human long non-coding RNAs with accurate 5' ends*](https://www.nature.com/articles/nature21374), Nature, 543(7644):199-204
* Rodríguez-Martín B, Palumbo E, Marco-Sola S, Griebel T, Ribeca P, Alonso G, Rastrojo A, Aguado B, Guigó R, **Djebali S**, 2017, [*ChimPipe: accurate detection of fusion genes and transcription-induced chimeras from RNA-seq data*](https://link.springer.com/article/10.1186/s12864-016-3404-9), BMC genomics, 18(1):7
* Breschi A, **Djebali S**, Gillis J, Pervouchine DD, Dobin A, Davis CA, Gingeras TR, Guigó R, 2016, [*Gene-specific patterns of expression variation across organs and species*](https://link.springer.com/article/10.1186/s13059-016-1008-y), Genome Biology, 17(1):151
* Teng M, Love MI, Davis CA, **Djebali S**, Dobin A, Graveley BR, Li S, Mason CE, Olson S, Pervouchine D, Sloan CA, Wei X, Zhan L, Irizarry RA, 2016, *A benchmark for RNA-seq quantification pipelines*, Genome Biology, 17:74
* Melé M, Ferreira PG, Reverter F, DeLuca DS, Monlong J, Sammeth M, Young TR, Goldmann JM, Pervouchine DD, Sullivan TJ, Johnson R, Segrè AV, **Djebali S**, Niarchou A, GTEx Consortium, Wright FA, Lappalainen T, Calvo M, Getz G, Dermitzakis ET, Ardlie KG, Guigó R, 2015, Human genomics. *The human transcriptome across tissues and individuals*, Science, 348(6235):660-5
* Pervouchine DD*, **Djebali S***, Breschi A*, Davis CA*, Barja PP, Dobin A, Tanzer A, Lagarde J, Zaleski C, See LH, Fastuca M, Drenkow J,	Wang H,	Bussotti G, Pei	B, Balasubramanian S, Monlong J, Harmanci A, Gerstein M, Beer MA, Notredame C, Guigó R, Gingeras TR, 2015, *Enhanced transcriptome maps from multiple mouse tissues reveal evolutionary constraint in gene expression*, Nature Communications, 6:5903
* Cheng Y, Ma	Z, ... **Djebali S** ..., Hardison RC, Snyder MP, 2014, *Principles of regulatory information conservation between mouse and human*, Nature, 515(7527):371-5
* Yue F*, Cheng Y*, Breschi A*, Vierstra J*, Wu W*, Ryba T*, Sandstrom R*, Ma Z*, Davis C*, Pope BD*, Shen Y*, Pervouchine DD, **Djebali S**, ... , Beer MA, Ren B, Mouse ENCODE Consortium, 2014, *A comparative encyclopedia of DNA elements in the mouse genome*, Nature, 515(7527):355-64
* Gerstein MB*, Rozowsky J*, Yan KK*, Wang D*, Cheng C*, Brown JB*, Davis CA*, Hillier L*, Sisu C*, Li JJ*, Pei B*, Harmanci AO*, Duff MO*, **Djebali S**, ... , Gingeras TR, Waterston R, 2014, *Comparative analysis of the transcriptome across distant species*, Nature, 512(7515):445-8
* Ferreira PG, Jares P, Rico D, Gomez-López G, Martínez-Trillos A, Villamor N, Ecker S, González-Pérez A, Knowles DG, Monlong J, Johnson R, Quesada V, **Djebali S**, Papasaikas P, López-Guerra M, Colomer D, Royo C, Cazorla M, Pinyol M, Clot G, Aymerich M, Rozman M, Kulis M, Tamborero D, Gouin A, Blanc J, Gut M, Gut I, Puente XS, Pisano DG, Martin-Subero JI, López-Bigas N, López-Guillermo A, Valencia A, López-Otín C, Campo E, Guigó R, 2014, *Transcriptome characterization by RNA sequencing identifies a major molecular and clinical subdivision in chronic lymphocytic leukemia*, Genome Research, 2:212-26
* Steijger T, Abril JF, Engström PG, Kokocinski F, Hubbard TJ, Guigó R, Harrow J, Bertone P; RGASP Consortium including **Djebali S**, 2013, *Assessment of transcript reconstruction methods for RNA-seq*, Nature Methods, 12:1177-84
* Durán E, **Djebali S**, González S, Flores O, Mercader JM, Guigó R, Torrents D, Soler-López M, Orozco M, 2013, *Unravelling the hidden DNA structural/physical code provides novel insights on promoter location*, Nucleic Acids Research 41(15):7220-30
* Derrien T*, Johnson R*, Bussotti G, Tanzer A, **Djebali S**, Tilgner H, Guernec G, Martin D, Merkel A, Knowles DG, Lagarde J, Veeravalli L, Ruan X, Ruan Y, Lassmann T, Carninci P, Brown JB, Lipovich L, Gonzalez JM, Thomas M, Davis CA, Shiekhattar R, Gingeras TR, Hubbard TJ, Notredame C, Harrow J, Guigó R, 2012, *The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression*, Genome Research, 9:1775-89
* Cheng C, Alexander R, Min R, Leng J, Yip KY, Rozowsky J, Yan KK, Dong X, **Djebali S**, Ruan Y, Davis CA, Carninci P, Lassman T, Gingeras TR, Guigó R, Birney E, Weng Z, Snyder M, Gerstein M, 2012, *Understanding transcriptional regulation by integrative analysis of transcription factor binding data*, Genome Research, 9:1658-67
* Tilgner H, Knowles DG, Johnson R, Davis CA, Chakrabortty S, **Djebali S**, Curado J, Snyder M, Gingeras TR, Guigó R, 2012, *Deep sequencing of subcellular RNA fractions shows splicing to be predominantly co-transcriptional in the human genome but inefficient for lncRNAs*, Genome Research, 9:1616-25
* **Djebali S***, Davis CA*, Merkel A, Dobin A, Lassmann T, Mortazavi A, Tanzer A, Lagarde J, Lin W, Schlesinger F, Xue C, Marinov GK, Khatun J, Williams BA, Zaleski C, Rozowsky J, Röder M, Kokocinski F, Abdelhamid RF, Alioto T, Antoshechkin I, Baer MT, Bar NS, Batut P, Bell K, Bell I, Chakrabortty S, Chen X, Chrast J, Curado J, Derrien T, Drenkow J, Dumais E, Dumais J, Duttagupta R, Falconnet E, Fastuca M, Fejes-Toth K, Ferreira P, Foissac S, Fullwood MJ, Gao H, Gonzalez D, Gordon A, Gunawardena H, Howald C, Jha S, Johnson R, Kapranov P, King B, Kingswood C, Luo OJ, Park E, Persaud K, Preall JB, Ribeca P, Risk B, Robyr D, Sammeth M, Schaffer L, See LH, Shahab A, Skancke J, Suzuki AM, Takahashi H, Tilgner H, Trout D, Walters N, Wang H, Wrobel J, Yu Y, Ruan X, Hayashizaki Y, Harrow J, Gerstein M, Hubbard T, Reymond A, Antonarakis SE, Hannon G, Giddings MC, Ruan Y, Wold B, Carninci P, Guigó R, Gingeras TR, 2012, *Landscape of transcription in human cells*, Nature, 489(7414):101-8
* ENCODE Project Consortium including **Djebali S**, 2012, *An integrated encyclopedia of DNA elements in the human genome*, Nature, 489(7414):57-74
* Dong X, Greven MC, Kundaje A, **Djebali S**, Brown JB, Cheng C, Gingeras TR, Gerstein M, Guigó R, Birney E, Weng Z, 2012, *Modeling gene expression using chromatin features in various cellular contexts*, Genome Biology, 13(9):R53
* Mouse ENCODE Consortium including **Djebali S**, 2012, *An encyclopedia of mouse DNA elements (Mouse ENCODE)*, Genome Biology, 13(8):418.
* **Djebali S***, Lagarde J*, Kapranov P*, Lacroix V*, Borel C, Mudge JM, Howald C, Foissac S, Ucla C, Chrast J, Ribeca P, Martin D, Murray RR, Yang X, Ghamsari L, Lin C, Bell I, Dumais E, Drenkow J, Tress ML, Gelpí JL, Orozco M, Valencia A, van Berkum NL, Lajoie BR, Vidal M, Stamatoyannopoulos J, Batut P, Dobin A, Harrow J, Hubbard T, Dekker J, Frankish A, Salehi-Ashtiani K, Reymond A, Antonarakis SE, Guigó R, Gingeras TR, 2012, *Evidence for transcript networks composed of chimeric RNAs in human cells*, PLoS One, 7(1):e28213
* ENCODE Project Consortium including **Djebali S**, 2011, *A user's guide to the encyclopedia of DNA elements (ENCODE)*, PLoS Biology,4:e1001046
* **Djebali S***, Kapranov P*, Foissac S*, Lagarde J*, Reymond A, Ucla C, Wyss C, Drenkow J, Dumais E, Murray RR, Lin C, Szeto D, Denoeud F, Calvo M, Frankish A, Harrow J, Makrythanasis P, Vidal M, Salehi-Ashtiani K, Antonarakis SE, Gingeras TR, Guigó R, 2008, *Efficient targeted transcript discovery via array-based normalization of RACE libraries*, Nature Methods, 7:629-35  
* **Djebali S**, Delaplace F, Roest Crollius H, 2006, *Exogean: a framework for annotating protein-coding genes in eukaryotic genomic DNA*, Genome Biology, 7 Suppl 1:S7.1-10


