[Courses](#courses)

[Supervision](#supervision)

## Courses <a name="courses"></a>
I have given the following courses:
<table>
  <thead>
      <tr>
      <th width=100px>Years</th>
      <th width=100px>Number of hours</th>
      <th width=400px>Topic</th>
      <th width=200px>Audience</th>
      <th width=200px>Location</th>
    </tr>
  </thead>
  <tbody>
     <tr>
      <td>2025</td>
      <td>2h</td>
      <td>Functional annotation databases and their interrogation</td>
      <td>Purpan Bioinfo group</td>
      <td>Toulouse</td>
    </tr>
    <tr>
      <td>2024</td>
      <td>2h</td>
      <td>Functional annotation databases and their interrogation</td>
      <td>INSERM workshop attendees</td>
      <td>Bordeaux</td>
    </tr>
    <tr>
      <td>2024</td>
      <td>2h</td>
      <td>Causal variant fine-mapping using functional annotations</td>
      <td>INSERM workshop attendees</td>
      <td>Bordeaux</td>
    </tr>
    <tr>
      <td>2022</td>
      <td>12h</td>
      <td>Linux and the command line</td>
      <td>Bioinformatics Master</td>
      <td>UPS, Toulouse</td>
    </tr>
    <tr>
      <td>2021-2022</td>
      <td>4h</td>
      <td>Functionnal annotation and common diseases</td>
      <td>Bioengineering Master</td>
      <td>UPS, Toulouse</td>
    </tr>
    <tr>
      <td>2012-2020</td>
      <td>72h</td>
      <td>Theoretical and practical course about ENCODE and/or the analysis of RNA-seq data</td>
      <td>Researchers and engineers</td>
      <td>Pasteur/EMBO/CRG/SIB</td>
    </tr>
    <tr>
      <td>2011</td>
      <td>8h</td>
      <td>Theoretical and practical computer science courses (UNIX and programming)</td>
      <td>Researchers in Biology</td>
      <td>CRG, Barcelona</td>
    </tr>
    <tr>
      <td>2002-2006</td>
      <td>280h</td>
      <td>Theoretical and practical computer science and bioinformatics courses</td>
      <td>License and Master students</td>
      <td>Evry University</td>
    </tr>
  </tbody>
</table>

## Supervision <a name="supervision"></a>
During my career, I have (co-)supervised the following people:
<table>
  <thead>
    <tr>
      <th width=150px>Name</th>
      <th width=450px>Project</th>
      <th width=100px>Level</th>
      <th width=100px>Supervision ratio</th>
      <th width=200px>Dates</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Amel Benarbia</td>
      <td>Characterizing genomic rearrangements at transcribed loci following double stranded breaks</td>
      <td>M2</td>
      <td>50%</td>
      <td>January to July 2025</td>
    </tr>
    <tr>
      <td>Victor Lefebvre</td>
      <td>Divergence of duplicated genes' cis-regulatory landscape</td>
      <td>M2 and now phD</td>
      <td>33%</td>
      <td>Since January 2024</td>
    </tr>
    <tr>
      <td> Océane Carpentier</td>
      <td>Knowledge graphs to identify miRNA/gene relationships</td>
      <td>M2</td>
      <td>50%</td>
      <td>January to July 2024</td>
    </tr>
    <tr>
      <td>Martin Racoupeau</td>
      <td>Comparison of methods for microRNA identification</td>
      <td>M2</td>
      <td>50%</td>
      <td>January to July 2023</td>
    </tr>
    <tr>
      <td>Zoé Gerber</td>
      <td>Identification of causal variants in coronary artery disease</td>
      <td>M2</td>
      <td>100%</td>
      <td>September 2022 to June 2023</td>
    </tr>
    <tr>
      <td>Tess Azevedo</td>
      <td>Gene co-expression networks and module identification in livestock species</td>
      <td>M1</td>
      <td>33%</td>
      <td>June to October 2022</td>
    </tr>
    <tr>
      <td>Smahane Chalabi</td>
      <td>Impact of the maternal diet's fiber content on the transcriptomes and epigenomes of fetus and piglets</td>
      <td>Postdoc</td>
      <td>50%</td>
      <td>October 2021 to March 2023</td>
    </tr>
    <tr>
      <td>Tristan Hoellinger</td>
      <td>Genome-wide identification of enhancer/gene relationships</td>
      <td>Master</td>
      <td>100%</td>
      <td>September 2020 to February 2022</td>
    </tr>
    <tr>
      <td>Pierre Guenzi-Tibéri</td>
      <td>Development of a Nextflow pipeline for quantifying the expression of small RNAs from high-throughput sequencing data</td>
      <td>M1</td>
      <td>50%</td>
      <td>June to August 2021</td>
    </tr>
    <tr>
      <td>Claire Martin</td>
      <td>Combining enhancer/gene relationships and genetic data to identify new variants and genes associated to Parkinson's disease</td>
      <td>Engineer</td>
      <td>50%</td>
      <td>September 2020 to March 2021</td>
    </tr>
    <tr>
      <td>Camille Mestre</td>
      <td>Genome-wide identification of enhancer/gene relationships</td>
      <td>M2</td>
      <td>60%</td>
      <td>January to July 2018</td>
    </tr>
    <tr>
      <td>Matthew Smart</td>
      <td>Comparison of chromatin accessibility and chromatin structure data</td>
      <td>L3</td>
      <td>70%</td>
      <td>March to September 2017</td>
    </tr>
    <tr>
      <td>Nikki Gentle</td>
      <td>Transcriptional Signatures of Viral Control in HIV-1 Infected South African Women</td>
      <td>PhD student</td>
      <td>30%</td>
      <td>January to July 2014</td>
    </tr>
    <tr>
      <td>Alessandra Breschi</td>
      <td>Human and mouse transcriptome comparison</td>
      <td>PhD student</td>
      <td>50%</td>
      <td>November 2012 to October 2015</td>
    </tr>
    <tr>
      <td>Bernardo Rodríguez Martín</td>
      <td>Chimeric transcript identification in vertebrate genomes</td>
      <td>M2 and PhD student</td>
      <td>100% and 30%</td>
      <td>January 2013 to October 2016</td>
    </tr>
    <tr>
      <td>Emilio Palumbo</td>
      <td>Robust pipeline development for the analysis of HTS functional data</td>
      <td>Engineer</td>
      <td>80%</td>
      <td>November 2012 to October 2015</td>
    </tr>
    <tr>
      <td>Matthieu Muffato</td>
      <td>Development of a user interface for the Exogean gene identification program</td>
      <td>M1</td>
      <td>80%</td>
      <td>April to July 2006</td>
    </tr>
    <tr>
      <td>Charles Vejnar</td>
      <td>Development of a pseudogene detection module for the Exogean gene identification program</td>
      <td>M1</td>
      <td>80%</td>
      <td>April to July 2005</td>
    </tr>
    <tr>
      <td>Khaled Yahiaoui</td>
      <td>Improvement and evaluation of Exogean, a eukaryotic gene identification program</td>
      <td>M2</td>
      <td>80%</td>
      <td>January to July 2005</td>
    </tr>
  </tbody>
</table>
